content/600-about/200-prisma-docs/30-docs-components/03-frontmatter.mdx

```diff
@@ -1,34 +1,53 @@
 ---
-title: Frontmatter
-metaDescription: 'Supported frontmatter variables.'
+title: Front matter
+metaDescription: Supported front matter variables.
 ---
 
 <TopBlock></TopBlock>
 
 ## <inlinecode>title</inlinecode>
 
-The `<h1>` title of the page.
+The `<h1>` title of the page. This title appears at the top of the page, and in the left-hand navigation.
+
+Note:
+
+- You can wrap the `title` text in single quotes, but this is not mandatory.
+- If you wrap the `title` text in single quotes, then you cannot include apostrophes in the text.
```

Comment:
You should be able to, you just need to escape it probably with a backslash?

Reply:
I did experiment with that a while back, at least in some of the other metadata fields, and I couldn't find a way to get it working